types/linkifyjs/tsconfig.json

```diff
@@ -1,25 +1,16 @@
 {
     "compilerOptions": {
         "module": "commonjs",
-        "lib": [
-            "es6",
-            "dom"
-        ],
+        "lib": ["es6", "dom"],
         "noImplicitAny": true,
         "noImplicitThis": true,
         "strictNullChecks": true,
         "strictFunctionTypes": true,
         "baseUrl": "../",
-        "typeRoots": [
-            "../"
-        ],
+        "typeRoots": ["../"],
         "types": [],
         "noEmit": true,
         "forceConsistentCasingInFileNames": true
     },
-    "files": [
-        "index.d.ts",
-        "html.d.ts",
-        "linkifyjs-tests.ts"
-    ]
+    "files": ["index.d.ts", "html.d.ts", "react.d.ts", "linkifyjs-tests.ts"]
 }
```

Comment:
Why was this file reformatted?

Reply:
I have VSCode configured to run Prettier on save, the line width is 80 chars so it reformatted this because it fits within a single line after I added `react.d.ts` to the list.

Reply:
Weird, I swear I was using prettier too… this is fine then!